src/assets/scripts/client/constants/aircraftConstants.js

```diff
@@ -203,6 +203,16 @@ export const PERFORMANCE = {
      */
     MAXIMUM_ANGLE_CONSIDERED_ESTABLISHED_ON_APPROACH_COURSE: 0.0872665, // appx. 5 degrees
 
+    /**
+     * Maximum angular difference from the hold inbound heading to consider the aircraft close
```

Comment:
Isn't that the outbound heading you've been dealing with?

Reply:
Ah, for the love of $DEITY, You wouldn't believe how many times I tried proof reading a diff that changes so few lines...